packages/plugin-core/src/utils/md.ts

```diff
@@ -494,9 +494,10 @@ export async function findReferencesById(id: string) {
     return;
   }
 
+  const engineNotes = await engine.findNotes({ excludeStub: true });
   const notesWithRefs = NoteUtils.getNotesWithLinkTo({
     note,
-    notes: engine.notes,
+    notes: engineNotes,
```

Comment:
We should move this link finding logic to the backend (later).  Maybe need to check on performance in `nightly` to see if it's acceptable - does backlinks panel still work in org-workspace?